src/Raven.Server/Config/Categories/SecurityConfiguration.cs

```diff
@@ -47,6 +47,11 @@ public class SecurityConfiguration : ConfigurationCategory
         [ConfigurationEntry("Security.Certificate.Exec", ConfigurationEntryScope.ServerWideOnly)]
         public string CertificateExec { get; set; }
 
+        [Description("A command or executable providing a .pfx certificate file. If specified, RavenDB will use HTTPS/SSL for all network activities. The certificate path setting takes precedence over executable configuration option.")]
+        [DefaultValue(null)]
+        [ConfigurationEntry("Security.Certificate.Exec.V2", ConfigurationEntryScope.ServerWideOnly)]
```

Comment:
I don't like this path name. It make sense, but it doesn't explain what the difference from the usual are.
It also doesn't handle users who are already using exec config, and we need to do something about it.

We can error (this is for 4.2) if this `Exec` alone is specified, but something there is required.

Reply:
Maybe something like, if the `Exec` is specified, and we need to load, we'll strip the first argument and call anyway.
And error on refresh / update ?

Reply:
Maybe you missed this but I added an exception to RavenServer.LoadCertificate which doesn't allow to use the old value. It tells them to switch to V2 and go to the documentation